java-client-model/src/main/java/com/cumulocity/rest/representation/tenant/GenerateStatisticsFileRequestRepresentation.java

```diff
@@ -0,0 +1,37 @@
+package com.cumulocity.rest.representation.tenant;
+
+import com.cumulocity.model.DateTimeConverter;
+import com.cumulocity.rest.representation.AbstractExtensibleRepresentation;
+import com.cumulocity.rest.representation.annotation.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.joda.time.DateTime;
+import org.svenson.JSONProperty;
+import org.svenson.converter.JSONConverter;
+
+import static com.cumulocity.rest.representation.annotation.Command.CREATE;
+
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class GenerateStatisticsFileRequestRepresentation extends AbstractExtensibleRepresentation {
```

Comment:
It is used in functional tests like many other cases which you correctly pointed out. I decided to put it here, but we can also think about coping them the `cucumber-java` only. Do you think it will be a better approach?

Reply:
I like last idea, move it only to cucumber-java, since external users of SDK should not touch it.